android/sdl_android/src/androidTest/java/com/smartdevicelink/test/TestValues.java

```diff
@@ -427,6 +430,13 @@
     public static final DynamicUpdateCapabilities GENERAL_DYNAMICUPDATECAPABILITIES = new DynamicUpdateCapabilities();
     public static final WindowState GENERAL_WINDOWSTATE = new WindowState();
 
+    public static final SeatOccupancy GENERAL_SEAT_OCCUPANCY = new SeatOccupancy();
+    public static final List<SeatStatus> GENERAL_SEATS_OCCUPIED = new ArrayList<SeatStatus>(1);
+    public static final JSONArray JSON_GENERAL_SEATS_OCCUPIED = new JSONArray();
```

Comment:
@kostyaBoss I would move lines 435 and 437 (`JSONArray` objects) to below line 563 where the other `JSONArray` declarations are.